### PR TITLE
feat: improve multi-conductor support

### PR DESCRIPTION
**Pull request by steveej**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -63,16 +62,17 @@ class EmergenceStress {
...
     return await EmergenceStress.withName(name)
   }
 
   static async withName (name) {
     console.log(`loading fresh page`)
 
     const browser = await puppeteer.launch({
-      headless: 'new'
+      headless: 'new',
+      executablePath: 'chromium'
```
Comment (steveej) [inline index.js]: this probably breaks on other systems. maybe an env var to set this would make sense. what's a good default?

Comment (neonphog) [reply index.js]: The default download is pretty small and cached. I'd say the default is a good default : ) - but sure, you can have an env to specify the executable path.

**Review by neonphog — APPROVED** [review]

**Review by jost-s — APPROVED** [review]
LGTM, but I haven't tested it

File: index.js (hunk 1 above)
Comment (steveej) [reply index.js]: the downwloaded binary didn't work for me. i just added a nix flake and added a env var to detect the executable dynamically. should be fine for now :blush:

**Review by neonphog — APPROVED** [review]

**Pull request merged**